learn-array-methods-by-building-a-statistics-calculator/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./styles.css" />
+    <script src="./script.js"></script>
+    <title>Statistics Calculator</title>
+  </head>
+  <body>
+    <h1>Statistics Calculator</h1>
+    <p>Enter a list of comma-separated numbers.</p>
```

Comment:
We might consider adding a short definition for mean, median and mode since some campers might not know or remember what they are.

Maybe we can add another p element or just explain it in the steps.

Reply:
I agree. Wouldn't hurt to have a short definition as another `p` element, or to use a `span` and a `title` attribute as a tooltip.